template-files/android/AndroidManifest.xml

```diff
@@ -98,15 +98,15 @@
 
     <activity
       android:name=".experience.ExperienceActivity"
-      android:configChanges="keyboard|keyboardHidden|orientation|screenSize"
+      android:configChanges="keyboard|keyboardHidden|orientation|screenSize|uiMode"
```

Comment:
Ok, so this change will have to be inspected from the perspective of `react-native-appearance` 🤔 
I'm not sure whether `react-native-appearance` can handle it properly (change from `recreating whole activity` to `fire some lifecycle method upon change`)? cc @brentvatne